projects/ccf-rui/src/styles.scss

```diff
@@ -27,7 +27,7 @@
   }
 
   .logo {
-    background-image: url('./assets/icons/default-logo.svg');
+    background-image: url('./assets/icons/hra-logo.svg');
```

Comment:
Is there a reason you change the urls instead of replacing the files?

Reply:
No particular reason. I can replace the files. Actually that makes more sense. I can do that.